File: EASSET.Web/Modules/ServerTypes/MasterData/UKPBColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { UKPBRow } from "./UKPBRow";

export interface UKPBColumns {
    UkpbNama: Column<UKPBRow>;
    UkpbKode: Column<UKPBRow>;
    UpbNama: Column<UKPBRow>;
    UnitNama: Column<UKPBRow>;
    SubUnitNama: Column<UKPBRow>;
    UkpbAlamat: Column<UKPBRow>;
}

export class UKPBColumns extends ColumnsBase<UKPBRow> {
    static readonly columnsKey = 'MasterData.UKPB';
    static readonly Fields = fieldsProxy<UKPBColumns>();
}

File: EASSET.Web/Modules/MasterData/UKPB/UKPBColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace EASSET.MasterData.Columns;

[ColumnsScript("MasterData.UKPB")]
[BasedOnRow(typeof(UKPBRow), CheckNames = true)]
public class UKPBColumns
{
    /*[EditLink, DisplayName("Db.Shared.RecordId"), AlignRigh]*/
    //public string Ukpbid { get; set; }
    [EditLink]
    public string UkpbNama { get; set; }
    public string UkpbKode { get; set; }
    public string UpbNama { get; set; }
    public string UnitNama { get; set; }
    public string SubUnitNama { get; set; }
    public string UkpbAlamat { get; set; }
    //public string ThAng { get; set; }
    //public string Op { get; set; }
    //public DateTime Lu { get; set; }
    //public string Pc { get; set; }
    //public short Dlt { get; set; }
}